packages/ui/src/ui/pages/navigation/Navigation/Navigation.js

```diff
@@ -77,6 +78,7 @@ function renderModals() {
             <TableMergeModal />
             <DynTablesStateModal />
             <LinkToModal />
+            <CreateACOModal />
```

Comment:
It looks like it is possible to create ACO objects only from `Navigation`, and I think it would be useful to allow to create new ACO from page of a query.

Reply:
We have figured out that creation of ACO from a query page requires new yt-api methods. Let's leave it as is for a while.